app/views/characters/_character_weapons.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="row characters-row">
+  <% @character_weapons.each do |character_weapon| %>
+  <div class="col-sm-4 characters-col">
+    <div class="card">
+      <div class="card-block">
+        <h4 class="card-title"><%= character_weapon.weapon.name %></h4>
+        <ul class="card-text">
+          <li>Damage: <%= character_weapon.damage %></li>
+          <li>Crit: <%= character_weapon.crit %></li>
+        </ul>
+        <%= link_to "Show", character_character_weapon_path(character_id: @character.id, id: character_weapon.id), class: "btn access_button" %>
```

Comment:
I think you can pass the objects directly instead of specifying the param names...